sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/core/CosmosTemplate.java

```diff
@@ -152,6 +150,14 @@ public CosmosTemplate(CosmosFactory cosmosFactory,
         this(cosmosFactory, cosmosConfig, mappingCosmosConverter, null);
     }
 
+    public String getDatabaseName() {
```

Comment:
why are these two APIs public? IMO, we can probably make them private if they are only used in this class.

Reply:
Done.